astropy/config/configuration.py

```diff
@@ -363,7 +363,7 @@ def reload(self):
 
         Returns
         -------
-        val
+        val : type
```

Comment:
I'm changing this to "val : object", which is the least-informative, but always true, return type.
I felt I had to put something there because the NumPyDocs style guide — to which X-ref adheres — says all Return values must specify the type.

Reply:
✅